### PR TITLE
Show label in empty load game screen

### LINKED ISSUE #2748: Load game screen is totally empty without saves.
Without a save to load, the load game screen is totally empty. This looks very weird.
There should at least be a message saying no saves to load.
Or maybe the load game button should be disabled.

### PR DESCRIPTION
**Pull request by lordiii**
**Brief Description of What This PR Does**

Implements a label to be displayed if no saves are found.
Also adds a translatable string related to that.

**Related Issues**

Closes #2748 

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author.

### CODE REVIEW

File: src/saving/SaveList.tscn
Comment (Kasterisk) [inline src/saving/SaveList.tscn:107]: This may be unintended change.

File: src/saving/SaveList.tscn
Comment (Kasterisk) [inline src/saving/SaveList.tscn:52]: Having the text vertically aligned to the center of the save list might be better.

**Review by Kasterisk — CHANGES_REQUESTED** [review]

**Review by RacerBG — COMMENTED** [review]
After applying the changes which Kasterisk suggested this PR will be good to go. It's quite clever implementation.

File: src/saving/SaveList.tscn
Comment (lordiii) [reply src/saving/SaveList.tscn:107]: Godot seems to remove any lines containing `visible = false`.
Cause might be Godot interpreting missing `visible`-values as false.
Re-Adding that in line in the SaveList.tscn file, results in Godot removing it.

Comment (hhyyrylainen) [reply src/saving/SaveList.tscn:107]: I've seen this type of change many times before. This is caused by setting a node that we have purposefully hidden, to be visible. The default value for a node is to be visible, meaning that when Godot removes this line, that's the same as making the node visible. So you need to in the Godot editor find this node and click it back to hidden state, or use git to revert this line.

Comment (lordiii) [reply src/saving/SaveList.tscn]: Node was already hidden, got removed anyway.
Used git to revert the line.

**Review by Kasterisk — APPROVED** [review]
LGTM. Thanks for contributing!

**Comment by hhyyrylainen** [comment]
This is good to merge once the translation merge conflicts are resolved. It may be easier to reset the changes to the translation files, merge master, and then run the localize script again, and put back in the English text.

**Comment by hhyyrylainen** [comment]
I took care of the merge conflict even though it turned out to be very simple (just the POT update header lines were conflicting).

**Review by hhyyrylainen — APPROVED** [review]
Looks good enough now with the small change I did. If I were to nitpick I'd extract the now a bit too indented for my taste block to a new method.

**Pull request merged**
